packages/libs/components/src/map/ChartMarker.tsx

```diff
@@ -23,6 +23,7 @@ export interface ChartMarkerProps
   borderWidth?: number;
   data: {
     value: number;
+    count?: number;
```

Comment:
It seems a shame to introduce this as an optional prop when our EDA usage depends on it (and requires the `count` prop in the `StandaloneMapMarkersResponse` io-ts type. In MapAnalysis we have to fall back to zero so it compiles:
`          count: (data.count ?? 0) + prevData[index].count,`


Of course adding as optional is the right thing to do for backwards compatibility. But we are only using it in storybook stories and the EDA, so let's see how much would need to be changed if we made it compulsory...  A lot of the storybook code would need `{value: 100, count: 100}` duplication (and the counts would be unused), and then also some changes would be needed in src/lib/core/hooks/mapMarkers.tsx and src/lib/map/analysis/MarkerConfiguration/CategoricalMarkerPreview.tsx

I guess the problem here is that we've converted `StandaloneMapMarkersResponse` data into `xxxMarkerProps` data well before it is used to create markers and _while we still need to do calculations on it_.

It seems like a fudge to add an optional prop to a low level component (in `components`) so that operations in higher level components (`eda`) can work, and that the lower-level component will never use.

So I think the solution should lie purely in the EDA code - but maybe check with @dmfalke because big changes are being made to the marker data code. Merge conflict alert!

If Dave's branch didn't exist, I'd suggest creating the following types - probably in `standaloneMapMarkers.tsx` 

```
interface DonutMarkerPropsWithCounts extends DonutMarkerProps {
  data: DonutMarkerProps['data'] & {
    count: number;
  }[];
}
interface ChartMarkerPropsWithCounts extends ChartMarkerProps {
  data: ChartMarkerProps['data'] & {
    count: number;
  }[];
}
```

And `useStandaloneMapMarkers` would return these types instead of the vanilla versions.

Reply:
What about combining the marker props in `useStandaloneMarkers`? This could be controlled by an option, maybe `summarizeMarkers: boolean`.

Reply:
I opted for @bobular's approach as it felt more straightforward in my hazy brain!